types/node-fetch/index.d.ts

```diff
@@ -14,6 +14,28 @@
 import { Agent } from "http";
 import { URLSearchParams } from "url";
 
+// This is defined here to prevent a dependency on a particular implementation
```

Comment:
```suggestion
// `Event` and `AbortSignal` are defined here to prevent a dependency on a particular implementation
```

Reply:
Just a thought, and this is definitely nitpicking, but how would you feel about moving these into an out-of-line file, e.g. `externals.d.ts` or similar, as they're not as such part of the node-fetch package, and including them in `tsconfig.json`?